release-notes.md

```diff
@@ -3,6 +3,27 @@
 See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) or [How to upgrade the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/upgrade-anbox-cloud-appliance/24186) for
 instructions on how to update your Anbox Cloud deployment.
 
+[Details=1.16.1]
+
+## 1.16.1 (December 15 2022)
```

Comment:
14 or 15?

Reply:
a placeholder. TBD.

Reply:
Lets put 14 as we're going to release today as promised.